tests/CommunityToolkit.Mvvm.UnitTests/Test_ObservableRecipient.cs

```diff
@@ -85,6 +92,23 @@ public void Test_ObservableRecipient_Broadcast(Type type)
         Assert.AreEqual(message.PropertyName, nameof(SomeRecipient<int>.Data));
     }
 
+    [TestMethod]
+    public void Test_IRecipient_VerifyTrimmingAnnotation()
```

Comment:
This test doesn't appear to be tied to `IRecipient` and is a 1:1 copy of `Test_ObservableValidator_VerifyTrimmingAnnotation`.

Reply:
Oh whoops, forgot to update this one. Fixed in https://github.com/CommunityToolkit/dotnet/pull/135/commits/5fb9f7039cef0621684b3580726d6e5c5c9891aa, thanks! 😄

Reply:
Thanks, Sergio. Though this test still doesn't seem to be related to `IRecipient` specifically. Maybe rename the method to clear up confusion?

```suggestion
    public void Test_IMessenger_VerifyTrimmingAnnotation()
```

Reply:
I think `IRecipient` is more appropriate because the generator specifically looks for that interface.
The fact you're using `IMessenger` is not really tied to the logic of the generator itself here.